deploy/charts/github-token-manager/values.yaml

```diff
@@ -0,0 +1,74 @@
+---
+## namespace: defaults to the namespace where the chart is deployed if not specified
+namespace: ~
+
+## crds
+##   install: true | false
+##     true: install the CRDs
+##     false: do not install the CRDs
+crds:
+  install: true
+
+## rbac
+##   create: true | false
+##     true: create the RBAC resources
+##     false: do not create the RBAC resources
+rbac:
+  install: true
+
+## service
```

Comment:
Probably worth clarifying here that this is the metrics service, probably even nesting under a `metrics:` top-level key…

Reply:
Done!